Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -425,8 +426,40 @@ private static void GetReadableAxeResults(AxeResultItem[] results, string type,
 
                     htmlAndSelector.InnerHtml = content.ToString();
                     htmlAndSelectorWrapper.AppendChild(htmlAndSelector);
+
+                    // Add fixes if this is for violations
+                    if (ResultType.Violations.Equals(type) && item.Any.Length > 0)
+                    {
+                        if (item.Any.Length == 1)
+                        {
+                            htmlAndSelector = doc.CreateTextNode("To fix this violation address the following issue:");
+                        }
+                        else
+                        {
+                            htmlAndSelector = doc.CreateTextNode("To fix this violation address at least one the following issues:");
+                        }
+
+                        htmlAndSelectorWrapper.AppendChild(htmlAndSelector);
+
+                        htmlAndSelector = doc.CreateElement("p");
+                        htmlAndSelector.SetAttributeValue("class", "wrapOne");
+                        htmlAndSelector.InnerHtml = $"{HttpUtility.HtmlEncode(item.Html)}";
+
+                        content = new StringBuilder();
+                        htmlAndSelector = doc.CreateElement("p");
+                        htmlAndSelector.SetAttributeValue("class", "wrapTwo");
+
+                        //Fix the following
+                        foreach (var allMessages in item.Any)
+                        {
+                            content.AppendLine($"Issue ({HttpUtility.HtmlEncode(allMessages.Impact)}): {HttpUtility.HtmlEncode(allMessages.Message)}<br>");
```

Comment:
It would be a little more a11y-friendly to use a list to format the list of issues, rather than line breaks

Reply:
Good call.